app/src/main/java/com/duckduckgo/app/bookmarks/ui/EditSavedSiteDialogFragment.kt

```diff
@@ -128,6 +139,29 @@ class EditSavedSiteDialogFragment : SavedSiteDialogFragment() {
         }
     }
 
+    private fun configureMenuItems() {
+        val toolbar = binding.savedSiteAppBar.toolbar
+        toolbar.menu.findItem(R.id.action_delete).isVisible = true
+        toolbar.menu.findItem(R.id.action_confirm_changes).isEnabled = false
+    }
+
+    override fun deleteConfirmationTitle(): String {
+        val isFavorite = (getSavedSite() as? Favorite != null)
```

Comment:
To avoid all these checks we should probably split this into an `EditFavoriteDialogFragment` and an `EditBookmarkDialogFragment` (But I think that can be separate task).